qa/tasks/cephfs/kernel_mount.py

```diff
@@ -196,7 +198,10 @@ def get_id_to_dir():
             ))
             raise
 
-    def _read_debug_file(self, filename):
+    def read_debug_file(self, filename):
```

Comment:
Yeah, sorry forgotting to mention, this PR will depend on https://github.com/ceph/ceph/pull/38037 and the change above is in one commit from https://github.com/ceph/ceph/pull/38037, in which it will make this method a public one.

Reply:
I see. Thanks.